packages/esm-patient-medications-app/src/api/api.ts

```diff
@@ -38,9 +52,7 @@ export function usePatientOrders(patientUuid: string, status: 'ACTIVE' | 'any')
   const drugOrders = useMemo(
     () =>
       data?.data?.results
-        ? data.data.results
-            .filter((order) => order.orderType.display === 'Drug Order')
-            ?.sort((order1, order2) => (order2.dateActivated > order1.dateActivated ? 1 : -1))
+        ? sortOrdersByDateActivated(data.data.results.filter((order) => order.orderType.display === 'Drug Order'))
```

Comment:
```suggestion
        ? sortOrdersByDateActivated(data.data.results)
```

We don't need to further filter orders by orderType. The `orderTypes` parameter in the request already guarantees that we're only getting back drug orders.